code/modules/clothing/suits/cloaks.dm

```diff
@@ -49,6 +49,30 @@
 	desc = "Worn by the Head of Personnel. It smells faintly of bureaucracy."
 	icon_state = "hopcloak"
 
+/obj/item/clothing/neck/cloak/gamer
+	name = "legendary gamer's cloak"
```

Comment:
for example, you could make this cloak's default name be "strange duped legendary gamer's cloak"

then when you earn it from actually getting the gaming skill up to legendary, you can set its name to be just "legendary gamer's cloak"

Reply:
or, alternatively, you can put that renaming thing that's called when you earn the cape or whenever you equip it

if someone equips the cape without having the corresponding legendary skill for it, its name returns to being the "fake" version. if a true legendary gamer dons the legendary gamer cape again, though, its name will return to being the "legit" version.